config/routes.rb

```diff
@@ -5,6 +5,9 @@
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   resources :offers, only: %i[index show] do
     resources :reviews, only: [:create]
+    resources :bookings, only: %i[new create]
```

Comment:
I think you need only create here. New is not being used.

Reply:
True, took it out while resolving a merge conflict on routes